test/integration/scene-stream-access/get-scene-stream-access-handler.spec.ts

```diff
@@ -0,0 +1,253 @@
+import { test } from '../../components'
+import { makeRequest, owner, admin, nonOwner } from '../../utils'
+import { TestCleanup } from '../../db-cleanup'
+import * as handlersUtils from '../../../src/controllers/handlers/utils'
+import { PlaceAttributes, StreamingAccessUnavailableError } from '../../../src/types'
+import { IngressInfo } from 'livekit-server-sdk/dist/proto/livekit_ingress'
+
+test('GET /scene-stream-access - gets streaming access for scenes', ({ components }) => {
+  const testPlaceId = `place-id-stream-access`
+  let cleanup: TestCleanup
+  const placeId = testPlaceId
+
+  type Metadata = {
+    identity: string
+    realmName: string
+    parcel: string
+    hostname: string
+    sceneId: string
+  }
+
+  let metadataLand: Metadata
+  let metadataWorld: Metadata
+  let mockIngress: IngressInfo
+  let mockSceneStreamAccess: any
+
+  beforeEach(async () => {
+    cleanup = new TestCleanup(components.database)
+
+    mockIngress = {
+      id: 'mock-ingress-id',
+      name: 'mock-ingress',
+      url: 'rtmp://mock-stream-url',
+      streamKey: 'mock-stream-key',
+      ingressId: 'mock-ingress-id'
+    } as IngressInfo
+
+    mockSceneStreamAccess = {
+      id: 'mock-access-id',
+      place_id: placeId,
+      streaming_url: 'rtmp://mock-stream-url',
+      streaming_key: 'mock-stream-key',
+      ingress_id: 'mock-ingress-id',
+      created_at: Date.now(),
+      active: true
+    }
+
+    metadataLand = {
+      identity: owner.authChain[0].payload,
+      realmName: 'test-realm',
+      parcel: '10,20',
+      hostname: 'https://peer.decentraland.zone',
+      sceneId: 'test-scene'
+    }
+
+    metadataWorld = {
+      identity: owner.authChain[0].payload,
+      realmName: 'name.dcl.eth',
+      parcel: '20,20',
+      hostname: 'https://worlds-content-server.decentraland.org/',
+      sceneId: 'test-scene'
+    }
+
+    jest.spyOn(handlersUtils, 'validate').mockResolvedValue(metadataLand)
+    jest.spyOn(components.sceneFetcher, 'getPlace').mockResolvedValue({
+      id: placeId,
+      positions: ['10,20'],
+      owner: owner.authChain[0].payload
+    } as PlaceAttributes)
+    jest.spyOn(components.sceneFetcher, 'hasLandPermission').mockResolvedValue(true)
+    jest.spyOn(components.sceneFetcher, 'hasWorldPermission').mockResolvedValue(false)
+    jest.spyOn(components.sceneAdminManager, 'isAdmin').mockResolvedValue(false)
+    jest.spyOn(components.livekit, 'getSceneRoomName').mockReturnValue(`test-realm:test-scene`)
+    jest.spyOn(components.livekit, 'getWorldRoomName').mockReturnValue(`name.dcl.eth`)
+    jest.spyOn(components.livekit, 'getOrCreateIngress').mockResolvedValue(mockIngress)
+    jest.spyOn(components.sceneStreamAccessManager, 'getAccess').mockResolvedValue(mockSceneStreamAccess)
+    jest.spyOn(components.sceneStreamAccessManager, 'addAccess').mockResolvedValue(undefined)
```

Comment:
As these are integration tests, and are practically the only good way we have to test the DB integration, what do you think about not mocking them and doing tests using the DB?